dts/arm/ti/am62x_sk_m4.dtsi

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2023 Texas Instruments Incorporated
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <freq.h>
+#include <arm/armv7-m.dtsi>
+#include <zephyr/dt-bindings/pinctrl/ti-k3-pinctrl.h>
+
+/ {
+
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu0: cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m4f";
+			reg = <0>;
+		};
+	};
+
+	sram0: memory@0 {
+		compatible = "mmio-sram";
+		reg = <0x0 DT_SIZE_K(192)>;		/* 192 KB of SRAM (I-Code) */
+	};
+
+	sram1: memory1@40000 {
+		compatible = "mmio-sram";
+		reg = <0x40000 DT_SIZE_K(64)>;		/* 64 KB of SRAM (D-Code) */
+	};
+
+	sysclk: system-clock {
+		compatible = "fixed-clock";
+		clock-frequency = <400000000>;
+		#clock-cells = <0>;
+	};
+
+	pinctrl: pinctrl@4080000 {
+		compatible = "ti,k3-pinctrl";
+		reg = <0x04080000 0x8000>;
```

Comment:
The datasheet defines `MCU_PADCONFIG` as starting from `0x04084000`.
Other projects (like Linux and U-Boot) use `0x04084000 0x88`, could this be kept consistent with those device trees?

Reply:
I have fixed this. It is now in accordance with the Linux DT. Sorry for the confusion.